aspnetcore/blazor/debug.md

```diff
@@ -44,17 +44,53 @@ Debugging requires either of the following browsers:
 
 ## Procedure
 
-1. Run a Blazor WebAssembly app in `Debug` configuration. Pass the `--configuration Debug` option to the [dotnet run](/dotnet/core/tools/dotnet-run) command: `dotnet run --configuration Debug`.
-1. Access the app in the browser.
-1. Place the keyboard focus on the app, not the developer tools panel. The developer tools panel can be closed when debugging is initiated.
+# [Visual Studio](#tab/visual-studio)
+
+1. Run a Blazor WebAssembly app in `Debug` configuration without debugging (**Ctrl**+**F5** instead of **F5**).
+1. Open the Debug properties (last entry in the **Debug** menu) of the app and copy the HTTP **App URL**. Browse to the HTTP address (not the HTTPS address) of the app using a Chromium based browser (Edge Beta or Chrome).
+1. Place the keyboard focus on the app in the browser window, not the developer tools panel. It's best to keep the developer tools panel closed for this procedure. After debugging has started, you can re-open the developer tools panel.
 1. Select the following Blazor-specific keyboard shortcut:
    * `Shift+Alt+D` on Windows/Linux
    * `Shift+Cmd+D` on macOS
-1. Follow the steps listed on the screen to restart the browser with remote debugging enabled.
-1. Select the following Blazor-specific keyboard shortcut once again to start the debug session:
+
+   If you receive the **Unable to find debuggable browser tab**, see [Enable remote debugging](#enable-remote-debugging).
+   
+   After enabling remote debugging:
+   
+   1\. A new browser window opens. Close the prior window.
```

Comment:
It doesn't look like these are rendering as a nested numbered list:

![image](https://user-images.githubusercontent.com/1874516/72570991-3a603700-3873-11ea-95f0-04b22f8c8624.png)

Reply:
🐀🍔 *Roasted Rat Burgers!*:tm: 🍔🐀

That should've worked. Let me add a line after the last one there on the next commit, and you can let me know if it worked.

Reply:
btw wrt ...

> work is in progress to enable debugging with Visual Studio

These steps worked here. It seems "enabled" ... just not perhaps "fully" enabled.

Reply:
> *Early* support exists for debugging Blazor WebAssembly using the browser dev tools in Chromium-based browsers (Chrome/Edge), and work is in progress to fully enable debugging with Visual Studio and enable debugging with Visual Studio Code.

Reply:
Line too long ... how about ...

> *Early* support exists for debugging Blazor WebAssembly using the browser dev tools in Chromium-based browsers (Chrome/Edge). Work is in progress to:
>
> * Fully enable debugging in Visual Studio.
> * Enable debugging with Visual Studio Code.

Reply:
That sounds fine. The point here is that most people will expect to just F5 in VS. That's the experience we plan to ship. The process of manually attaching VS is just something you can try out for now until the work on F5 debugging is done.